docs/user/reference/metrics/timing_distribution.md

```diff
@@ -239,6 +239,101 @@ Glean.pages.pageLoad.stopAndAccumulate(timerId);
 
 {{#include ../../../shared/tab_footer.md}}
 
+### `accumulateSamples`
+
+Accumulates the provided signed samples in the metric.
+This is required so that the platform-specific code can provide us with
+64 bit signed integers if no `u64` comparable type is available. This
+will take care of filtering and reporting errors for any provided negative
+sample.
+
+Please note that this assumes that the provided samples are already in
+the "unit" declared by the instance of the metric type (e.g. if the
+instance this method was called on is using `TimeUnit::Second`, then
+`samples` are assumed to be in that unit).
+
+This discards any negative value in `samples` and report an `ErrorType::InvalidValue`
+for each of them. Reports an `ErrorType::InvalidOverflow` error for samples that
+are longer than maximum sample time for the given unit.
```

Comment:
These should be in the `Limits` and `Recorded errors` section (like https://mozilla.github.io/glean/book/reference/metrics/custom_distribution.html#accumulatesamples )